peps/pep-0796.rst

```diff
@@ -0,0 +1,264 @@
+PEP: 796
+Title: Relative Virtual Environment Home
+Author: Richard Levasseur <richardlev@gmail.com>
+Sponsor: Alyssa Coghlan <ncoghlan@gmail.com>
+Discussions-To: Pending
+Status: Draft
+Type: Standards Track
+Created: 26-Jun-2025
+Python-Version: 3.15
+
+.. highlight:: rst
+
+
+Abstract
+========
+
+This PEP describes how a relative path for ``home`` in a Python virtual
+environment's ``pyvenv.cfg`` is understood by the Python startup process.
+Specifically, how it is canonicalized into an absolute path later used
+by the runtime. This small detail is a fundamental building block for
+virtual environments to be more portable.
+
+Motivation
+==========
+
+There are two main motivations for allowing relative paths in ``pyvenv.cfg``.
+
+First, it is currently prescribed that the ``home`` value in ``pyvenv.cfg`` be
+an absolute path. The behavior of relative paths is unspecified. While
+techniques exist to work around this for every other sub-part of a virtual
+environment, the one remaining part without a tenable solution is how the
+Python runtime itself finds ``PYTHONHOME``. This is because, currently, the
+startup process requires absolute paths be used for the ``home`` key in
+``pyvenv.cfg``. If a relative path is used, behavior is unspecified (the
+current implementation ends up making it relative to the process's current
+working directory, making it untenable to use).
+
+This requirement is overly proscriptive and restrictive because, given a known
+anchor point, it's easy to transform a relative path to an absolute path and
+still retain predictable and reliable behavior. Thus, the absolute path
+requirement should be relaxed and relative path behavior allowed and defined.
+
+Second, such relative paths are a building block to enable portable virtual
```

Comment:
It'd be nice to hear more about _why_ and _how_ a relative path makes the virtual environment more portable. A Python installation will still need to exist in a specific location on the new host, right? Is the pitch for a relative path that you'd copy a virtual environment _and_ the Python installation together?

Reply:
I added a sentence to help answer this question. PTAL. Or were you thinking more like a dedicated section?